.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "restructuredtext.confPath": ""
+}
```

Comment:
It seems like this was added here by mistake. It shouldn't go here.

Reply:
Oh true, sorry. Removed

Reply:
No problem. I always got the same issue :sweat_smile: